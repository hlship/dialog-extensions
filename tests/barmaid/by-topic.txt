ask barmaid about rumors
ask barmaid about eternal life
undo
ask barmaid about immortality
undo
ask barmaid about immortal
codger
ask barmaid about frobozz
tell barmaid about plugh
talk to barmaid about snert

