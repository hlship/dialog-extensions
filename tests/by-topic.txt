ask barmaid about rumors
%% ask barmaid about eternal life
ask barmaid about immortality
dgt