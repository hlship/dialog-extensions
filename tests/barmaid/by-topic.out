

It is a long and riotous evening, full of unlikely stories and tall tales. But
now, most of the patrons have gone away to their rooms to sleep, or have passed
out before the fire. Even the two black bitch pups are curled on the hearth-
stone, snuffling through tiny wet noses, and pawing the air in sleep. Now is the
time to find out whether the rumors that brought you to this neighborhood are
true.

The Inn
The light is low and red. Above the fireplace hangs a map of this part of the
Empire, and on the adjacent wall is the crimson and silver banner of Richard the
Rose. The barmaid works her way from table to table, clearing away dishes and
mugs.

> ask barmaid about rumors
You say hello to the Bar Maid.

"Where I come from, over the black hills there, they say that men this side of
the mountain live as old as Methuselah," you remark. "They say the secret of
eternal life is here."

"Oh, do they?" she says, sweeping crumbs of cheese and crusty bread into her
hand. "The oldest codger around these parts is old Garrick, and I would't put
him beyond his four-score and ten."

You could ask where Garrick lives, ask where the barmaid comes from, or ask
whether she's heard the stories.

> ask barmaid about eternal life
"Have you heard any stories of long-living men?" you press her.

She purses her lips and scrubs at a circle-shaped stain on the table before her.
"If you're a fool come looking for a spring of life or a vein of immortal gold
buried in the black hills, you'd do better to go back home where you come from."

You could ask where Garrick lives or ask where the barmaid comes from.

> undo
Undoing the last turn (ask barmaid about eternal life).
The Inn

> ask barmaid about immortality
"Have you heard any stories of long-living men?" you press her.

She purses her lips and scrubs at a circle-shaped stain on the table before her.
"If you're a fool come looking for a spring of life or a vein of immortal gold
buried in the black hills, you'd do better to go back home where you come from."

You could ask where Garrick lives or ask where the barmaid comes from.

> undo
Undoing the last turn (ask barmaid about immortality).
The Inn

> ask barmaid about immortal
"Have you heard any stories of long-living men?" you press her.

She purses her lips and scrubs at a circle-shaped stain on the table before her.
"If you're a fool come looking for a spring of life or a vein of immortal gold
buried in the black hills, you'd do better to go back home where you come from."

You could ask where Garrick lives or ask where the barmaid comes from.

> codger
"Where does this old Garrick live?" you ask, trying not to seem too eager.

"Down at the pig farm," she says. "The turn off is just before you come to the
river, on the main road west. You can find it by the smell."

You could ask where the barmaid comes from.

> 
